INDEX

```diff
@@ -5,3 +5,4 @@ x86_64/vmlinuz-5.5.0	https://libbpf-ci.s3-us-west-1.amazonaws.com/x86_64/vmlinuz
 x86_64/vmlinuz-4.9.0	https://libbpf-ci.s3-us-west-1.amazonaws.com/x86_64/vmlinuz-4.9.0
 s390x/libbpf-vmtest-rootfs-2022.10.23-bullseye-s390x.tar.zst	https://libbpf-ci.s3.us-west-1.amazonaws.com/libbpf-vmtest-rootfs-2022.10.23-bullseye-s390x.tar.zst
 aarch64/libbpf-vmtest-rootfs-2022.10.23-bullseye-arm64.tar.zst	https://libbpf-ci.s3.us-west-1.amazonaws.com/libbpf-vmtest-rootfs-2022.10.23-bullseye-arm64.tar.zst
+ppc64le/libbpf-vmtest-rootfs-2023.12.12-bullseye-ppc64el.tar.zst    https://dabbasa.blob.core.windows.net/ppc64le/libbpf-vmtest-rootfs-2023.12.12-bullseye-ppc64el.tar.zst
```

Comment:
Do you mind removing this change? I can build and upload an image for ppc64el if you need.

Reply:
@chantra Thats great! Please upload image for ppc64el. I was testing on same branch so the change reflected here. removing same.